bundles/org.openhab.binding.mqtt.generic/src/main/resources/ESH-INF/config/trigger-channel-config.xml

```diff
@@ -20,9 +19,9 @@
 			<label>Incoming Value Transformations</label>
 			<description><![CDATA[
 			Applies transformations to an incoming MQTT topic value.
-			A transformation example for a received JSON would be "JSONPATH:$.device.status.temperature" for
-			a json {device: {status: { temperature: 23.2 }}}.
-
+			This can be used to map the events sent by the device to common values for all devices using,
+			for example, the MAP transformation 
```

Comment:
```suggestion
			e.g. the MAP transformation.
```

Reply:
Sorry, had just deleted my first suggestion and replaced it while you were merging it 😟. Never mind, all fine now, thanks.